messaging.rst

```diff
@@ -352,6 +345,6 @@ In case a Raiden node goes offline or does not send the final balance proof to i
 
 **Limit to number of simultaneously pending transfers**
 
-The number of simultaneously pending transfers per channel is limited. The client will not initiate, mediate or accept a further pending transfer if the limit is reached. This is to avoid the risk of not being able to unlock the transfers, as the gas cost for this operation grows with the size of the Merkle tree and thus the number of pending transfers.
+The number of simultaneously pending transfers per channel is limited. The client will not initiate, mediate or accept a further pending transfer if the limit is reached. This is to avoid the risk of not being able to unlock the transfers, as the gas cost for this operation grows with the size of the pending lock list and thus the number of pending transfers.
```

Comment:
Same here. Please ignore this comment if you do indeed intent to call it "pending lock list" and not "pending locks list"

Reply:
Same here.

Reply:
Thanks. There are more places with "pending lock list" please change all of them now that there is a better term for it.

Reply:
Alright.

Reply:
Done all over.  I searched for "list lock" and I don't see anything.